Tools/autotest/arduplane.py

```diff
@@ -5312,6 +5312,37 @@ def SagetechMXS(self):
         home = self.home_position_as_mav_location()
         self.assert_distance(home, adsb_vehicle_loc, 0, 10000)
 
+    def MinThrottle(self):
+        '''Make sure min throttle does not apply in manual mode and does in FBWA'''
+        self.context_push()
```

Comment:
You get a free context-push just before the test starts, so you don't need a pair of these within your test (unless you need to reboot the vehicle after popping parameters)

Reply:
removed.